src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -130,7 +129,7 @@ public boolean isRegistered() {
     }
 
     public String getRegistrationUrl() {
-        return new Url(Config.APP_URL + Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
+        return new Url(Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
```

Comment:
Consider going to a link that needs redirect, in which the redirection path takes you to the registration URL with the original URL as the next URL. Let's say the link is `/page/xyz`. The registration URL previously will include the app URL, which makes it an absolute path, e.g `http://app.url/registration?next=/page/xyz`.
1. To make it work the app URL needs to be correct. This is against what is written on `build.properties` (`For dev server, any URL would do.`).
2. Absolute path is totally unnecessary unless for presentation purpose (e.g sending the URLs in emails). See the URL just right after that, the `getPublicProfilePictureUrl()`? That one is relative, it works, and it is clearly the better choice.